### PR TITLE
Add create/delete tag API test case

### PR DESCRIPTION
**Pull request by ywk253100**
Add create/delete tag API test case

Signed-off-by: Wenkai Yin <yinw@vmware.com>

### CODE REVIEW

File: tests/apitests/python/test_create_delete_tag.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        """
+        #1. Create a new user(UA);
+        TestProjects.user_id, user_name = self.user.create_user(user_password = self.user_password, **ADMIN_CLIENT)
+
+        TestProjects.USER_CLIENT=dict(with_tag = True, endpoint = self.url, username = user_name, password = self.user_password)
+
+        #2. Create a new project(PA) by user(UA);
+        TestProjects.project_id, TestProjects.project_name = self.project.create_project(metadata = {"public": "false"}, **TestProjects.USER_CLIENT)
+
+        #3. Push an image(IA) to Harbor by docker successfully;
```
Comment (renmaosheng) [inline tests/apitests/python/test_create_delete_tag.py]: do we need to add such test for helm and cnab as well? thanks.

**Comment by renmaosheng** [comment]
please link the issue# to be fixed. :)

**Comment by ywk253100** [comment]
Fixes https://github.com/goharbor/harbor/issues/10764

File: tests/apitests/python/test_create_delete_tag.py (hunk 1 above)
Comment (ywk253100) [reply tests/apitests/python/test_create_delete_tag.py]: The way how the tag API works is same for all kinds of artifacts, if it works for image, it should work for other artifacts as well.

**Pull request merged**